templates/zerver/help/restrict-editing.md

```diff
@@ -0,0 +1,50 @@
+# Restrict editing of old messages and topics
+
+If you are the administrator of a Zulip organization, you can easily change the
+time limit that your realm's users have to change their messages after sending
+them. Alternatively, you can choose to disable message editing for your realm
+users.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Administration** from the dropdown menu that appears.
+
+    ![Administration dropdown](/static/images/help/administration.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the
+**Administration** page. You will be located on the **Organization settings**
+tab by default.
+
+    ![Administration page](/static/images/help/admin-window.png)
+
+4. In the **Organizations settings** section, locate the **Users can edit old messages**
+checkbox and **Message edit limit in minutes (0 for no limit)** input field
+underneath it.
+
+    ![Message editing options](/static/images/help/message-editing-options.png)
+
+    By default, user message editing is enabled for 10 minutes after sending.
+
+    * **Users can edit old messages** - Uncheck this option if you wish to
+    disable message editing. Upon doing so, the **Message edit limit in minutes (0 for no limit)**
+    input field will be grayed out.
+
+        ![Message edit limit grayed out](/static/images/help/message-edit-limit-gray.png)
+
+    * **Message edit limit in minutes (0 for no limit)** - If you enable message
+    editing in your realm, you can restrict the time that realm users have to
+    edit their messages. Simply input the time limit in minutes that you would
+    like to set; for example, if you want to set a message edit limit of 5
+    minutes, enter **5** in the field.    
+
+        ![Message edit limit for 5 minutes](/static/images/help/message-edit-limit-5.png)
+
+    If you would like to disable the message editing time limit for your realm,
```

Comment:
I initially got the impression that by entering '0' you can disable the editing feature like you explained above. I guess you can mention that the time limit on edits is removed and users can edit their messages whenever they want.